cupy/creation/from_data.py

```diff
@@ -137,7 +139,16 @@ def copy(a, order='K'):
 # TODO(okuta): Implement frombuffer
 
 
-# TODO(okuta): Implement fromfile
+def fromfile(file, dtype=float, count=-1, sep='', offset=0):
```

Comment:
`offset` is new in NumPy 1.17.0, so users running older NumPy cannot use this option.
One idea may be to make the entire argument `*args, **kwargs` and just pass it to `numpy.fromfile`.

Reply:
SGTM

Reply:
Good catch!